deployment/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rust:1.56 as builder
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    clang \
+    libclang-dev \
+    libssl-dev \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /build/
+
+COPY . .
+
+RUN cargo install forc
```

Comment:
As per my other comment, this should be a multistage dockerfile. The first stage should run
`cargo build --release -p forc`.

Then afterwards, the resultant `forc` binary in the target directory should be copied into the second image stage which gets published.